README.md

```diff
@@ -41,7 +44,13 @@ This installer is incompatible with `default` namespace installations, install a
 
 ## Configuration
 
-For convenience you can configure your Red Hat Developer Hub, GitOps, and Pipelines Operators to enable the use of AI Software Templates by running the following:
+> [!IMPORTANT] 
```

Comment:
Should we have this section after `bash configure.sh`?

Otherwise ppl will venture off before running configure.sh 🤔

Reply:
I was thinking if it is before then they don't run `configure.sh` expecting it to just work for their pre-existing and then they have all the necessary info of what is going on at the start, wdyt? I'm fine either way.

Reply:
yeah its ok, I like your justification